pkg/gatherers/clusterconfig/gather_dvo_metrics.go

```diff
@@ -20,12 +20,14 @@ import (
 
 	"github.com/openshift/insights-operator/pkg/config"
 	"github.com/openshift/insights-operator/pkg/record"
+	"github.com/openshift/insights-operator/pkg/types"
 	"github.com/openshift/insights-operator/pkg/utils"
 	"github.com/openshift/insights-operator/pkg/utils/marshal"
 )
 
 const (
 	managedDVONamespaceName = "openshift-deployment-validation-operator"
+	serviceLabelSelector    = "name=deployment-validation-operator"
```

Comment:
I would rename this to 'serviceDVOLabelSelector` since it is declared at the global package scope and can be confusing.

Reply:
the constant is not exported